TreeGrid.php

```diff
@@ -135,12 +135,18 @@ class TreeGrid extends Widget {
     public $parentColumnName;
 
     /**
+     * @var string root of parent column used to build tree
+     */
+    public $parentRoot = null;
+
+    /**
      * @var array grid column configuration. Each array element represents the configuration
      * for one particular grid column.
      * @see \yii\grid::$columns for details.
      */
     public $columns = [];
 
+    public $collapsed = true;
```

Comment:
I think that it's not necessary, beacause it's an infomation of `pluginOptions`.
I think better to use:
```php
ArrayHelper::getValue($this->pluginOptions, 'initialState');
```